src/Domain/Fixture/FixtureCreator.php

```diff
@@ -437,6 +439,66 @@ private function verifySut(): void {
     }
   }
 
+  /**
+   * Composer require all the dev-dependencies of SUT.
+   *
+   * @see https://backlog.acquia.com/browse/ORCA-353
+   *
+   * @throws \Acquia\Orca\Exception\OrcaFileNotFoundException
+   * @throws \Acquia\Orca\Exception\OrcaParseError
+   * @throws \Acquia\Orca\Exception\OrcaException
+   */
+  private function composerRequireSutDevDependencies(): void {
+    $this->output->section('Adding dev-dependencies of SUT');
+    $dev_dependencies = [];
+    $package = $this->options->getSut();
+
+    if ($package === NULL) {
+      $this->output->writeln("No SUT defined");
+      return;
+    }
+    $subextensions = $this->subextensionManager->getByParent($package);
+
+    do {
+      $dev_dependencies[] = $this->computeDevDependenciesByPackage($package);
+      if (count($subextensions) <= 0) {
+        break;
+      }
+      $package = array_shift($subextensions);
+    } while (1);
+
+    $this->composer->requirePackages(array_values(array_filter($dev_dependencies)));
+
+  }
+
+  /**
+   * Computes the dev-dependencies of a given package.
+   *
+   * @param \Acquia\Orca\Domain\Package\Package $package
+   *   The Package in question.
+   *
+   * @return string|null
+   *   List of dev-dependencies found.
+   *
+   * @throws \Acquia\Orca\Exception\OrcaException
+   * @throws \Acquia\Orca\Exception\OrcaFileNotFoundException
+   * @throws \Acquia\Orca\Exception\OrcaParseError
+   */
+  private function computeDevDependenciesByPackage(Package $package): ?string {
+    $dev_dependencies =
+      $this->subextensionManager->findDevDependenciesByPackage($package);
+    if ($dev_dependencies === NULL || count($dev_dependencies) === 0) {
```

Comment:
I think this would be just slightly clearer:

```suggestion
    if ($dev_dependencies === NULL || $dev_dependencies === []) {
```